### PR TITLE
Finish show subject and modife a little layout

### PR DESCRIPTION
**Pull request by trungnn142**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -148,3 +148,28 @@ footer {
 ul {
   list-style-type: none;
 }
+
+fieldset.fsStyle {
+border: 1px solid #999999;
+padding: 4px;
+margin: 5px;
```
Comment (vinhnd2010) [inline app/assets/stylesheets/custom.css.scss]: indent

File: app/views/supervisor/subjects/_task_fields.html.erb [hunk 2]
```diff
@@ -1,10 +1,12 @@
 <div class="fields">
     <%= link_to_remove_fields t(".remove_task"), f %><br />
-    <div class="form-group">
-      <%= f.text_field :name, class: "form-control", placeholder: "Name" %>
-    </div>
-    <div class="form-group">
-      <%= f.number_field :order, class: "form-control", placeholder: "Position" %>
+    <div class="row">
+      <div class="col-lg-8">
+        <%= f.text_field :name, class: "form-control", placeholder: "Name" %>
```
Comment (ThuBM) [inline app/views/supervisor/subjects/_task_fields.html.erb]: I18n

File: app/views/supervisor/subjects/_task_fields.html.erb [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+      </div>
+      <div class="col-lg-2">
+        <%= f.number_field :order, class: "form-control", placeholder: "Position" %>
```
Comment (ThuBM) [inline app/views/supervisor/subjects/_task_fields.html.erb]: I18n

File: config/routes.rb [hunk 4]
```diff
@@ -13,4 +13,5 @@
   end
 
   resources :users
+  resources :user_subjects, only: [:show, :create, :update]
```
Comment (ThuBM) [inline config/routes.rb]: don't need this resources

File: app/views/supervisor/subjects/show.html.erb [hunk 5]
```diff
@@ -15,6 +15,18 @@
 </div>
 <div class="row">
   <div class="col-lg-12">
-    <p><%= @subject.instruction %></p>
+    <p><%= @subject.instruction %></p><br />
+    <fieldset class="fsStyle">
+      <legend class="legendStyle">
+        <span class="label label-info"><%= t ".Task"%></span>
```
Comment (ThuBM) [inline app/views/supervisor/subjects/show.html.erb]: add space before %>

**Comment by ThuBM** [comment]
OK

**Pull request merged**